tests/corePDEs/SideSetLaplacian/input_3D.yaml

```diff
@@ -1,79 +1,79 @@
 %YAML 1.1
 ---
 ANONYMOUS:
-  Debug Output: 
+  Debug Output:
     Write Solution to MatrixMarket: 0
-  Problem: 
+  Problem:
     Phalanx Graph Visualization Detail: 0
     Solution Method: Steady
     Name: Side Laplacian 3D
     Solve As Side Set Equation: true
-    Side Set Name: SideSet2
+    Side Set Name: SideSet4
```

Comment:
Just curious, what is the purpose of changing the side set and node set names here?

Reply:
I was debugging by printing GIDs/LIDs, and having it on the bottom sideset (z=0) made it easier, since those GIDs are the first added in a STK3D mesh. Otherwise I had to sit down, do the math, and make sure the GIDs reported were all on SideSet2 (which is y=0).